sdk/python/sawtooth_sdk/client/sdk_client.py

```diff
@@ -0,0 +1,68 @@
+# Copyright 2017 Intel Corporation
```

Comment:
The name of the module 'sdk_client' duplicates the namespace.  If we left it as-is, something like 'rest_api_wrapper_utils' would be an appropriate name.